pkg/cache/cache.go

```diff
@@ -79,6 +82,18 @@ func (c *serviceAccountCache) pop(name, namespace string) {
 	delete(c.cache, namespace+"/"+name)
 }
 
+// Dump cache contents for debugginqg
+func (c *serviceAccountCache) List() string {
+	c.mu.Lock()
```

Comment:
nit: could even be `RLock()/RUnlock()` as this is a RWMutex.

Reply:
Reusing the lock used for other cache operations such as get and set to ensure synchronization, instead of creating another one.